refactor: vale errors in marketer contribution guidelines (#414)
mautic/mautic-community-handbook: docs/contributing/marketer.rst

Marketer
########

As the world's first Open Source Marketing Automation platform, Mautic faces a unique challenges.

Marketing automation is a term unfamiliar to many, including most people in open source communities, who may not realize there are open source alternatives to the tools they use. Similarly, many people in the wider world are unaware of marketing automation's potential benefits for their organizations or that these advantages are accessible through open source software.

You can imagine, therefore, how many people actually know about Mautic.

That's why your contributions matter.

Getting started as a marketer
*****************************

> Mautic has a Marketing Team which you can read more about on the :doc:`Marketing Team </teams/marketing_team>` section.

Reviewer: Small wording adjustment.

```suggestion
Mautic has a Marketing Team, which you can learn more about in the :doc:`Marketing Team </teams/marketing_team>` section.
```

Work in the public domain
*************************

Unless there's significant reason not to, the default approach is openness and transparency. Work happens in the open, usually on the shared :xref:`Mautic Google Drive` or through the team Canva account. This ensures that if someone is unable to complete a task, it's easy for another contributor to pick up where they left off. It also means contributors can always find previous work if needed for future re-use. 

Please always ensure that you upload work at regular—ideally daily—intervals. Use the file prefix of WIP-filename to indicate that it's currently in progress.

Update regularly
****************

Please provide regular updates on the issue. If you can’t complete the task, add a comment on the issue or let your Team Lead know so someone else can take it over.

> The team understands that life happens and it's easy to take on too much. No judgment. Contributors respect each other by providing as much notice as possible when they're unable to fulfill an assigned task.

Reviewer: I think we can adjust this a bit to get as closer as possible to the original version.

```suggestion
Life can be unpredictable, and it’s easy to feel overwhelmed by trying to manage too much at once. There's no judgment here. The team's goal is to respect one another by giving as much notice as possible if someone can't complete an assigned task.
```
